### PR TITLE
Backend merge: Project restructure

### LINKED ISSUE #335: Deploy modifying only one file + env
There is currently a need to modify at least `core/.env` from the state that it is in in git to get a working production server. 

On the other sites `docker-compose.yml` file is also modified. The things that need to be changed in these files should be added to the environment variables or in some other way that doesn't require separate configs in production.

### LINKED ISSUE #338: Separate common and site specific frontend files
Make directories for templates and  static, inside those separate comon and site specifc (only DaTe to begin with) files into separate subfolders - common and date.

Make sure django can still read both the common and site-specific files.

### LINKED ISSUE #348: Clean up settings
The django settings file should be cleaned up and organized if we want to have different settings files for each websites

We should also identify what would be common config and what is site-specific.

### LINKED ISSUE #542: feat: Replace AA news hard-code
**Is your feature request related to a problem? Please describe.**
Currently AA news is hardcoded as a separate model.

**Describe the solution you'd like**
Maybe consider adding support for dynamic categories and have a default category that will show up on the homepage.

**Additional context**
Might be hard with migrations

### LINKED ISSUE #544: fix: One table for static page navs
**Is your fix request related to a problem?**  
Current implementation of static page navs is clunky with 2 tables for staticpages and urls

**Describe the solution you'd like**  
Only have 1 table for everything allowing you to freely reorder

**Additional context**  
Should have reorderable admin

IMO it's fine if we drop the static page special sauce and just have e.g. a lookup for if it's a static page as a backup.

### PR DESCRIPTION
**Pull request by jaspnas**
Adding support for running multiple websites from the same branch

### CODE REVIEW

File: core/settings/common.py [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+# https://docs.djangoproject.com/en/2.1/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'postgres',
+        'USER': 'postgres',
+        'PASSWORD': env('DB_PASSWORD', default=''),
+        'HOST': 'db',
+        'PORT': 5432,
```
Comment (andreedd) [inline core/settings/common.py]: These could maybe be env vars?

**Review by andreedd — COMMENTED** [review]
Daamn, nicee :D

File: core/settings/common.py (hunk 1 above)
Comment (jaspnas) [reply core/settings/common.py]: Still need to go over this several times, so very much WIP for now :D

**Review by andreedd — COMMENTED** [review]
I like this solution. Good job! Did not test, just read changes.

Probably worth checking the css var colors still at some point, either here or in a new issue. Might be that just changing the css root vars is not enough. I think I also did some other small changes for the other sites, not sure.

**Comment by jaspnas** [comment]
> Probably worth checking the css var colors still at some point, either here or in a new issue. Might be that just changing the css root vars is not enough. I think I also did some other small changes for the other sites, not sure.

Will be a problem for once we throw in frontend from other sites into the main branch. Best to limit the scope now so we get this merged

**Comment by jaspnas** [comment]
FYI for anyone reviewing, some statics may have the wrong path as the path has changed in django but not in the templates, please add a comment containing which static file doesnt load and which page it occurs on if you find anything

**Comment by jaspnas** [comment]
Should now have all the functionality changes fully complete.

**Comment by theoabw** [comment]
The style for the members/info table isn't dependent on the project css vars. Should be fixed at some point.

**Review by Theoabw — APPROVED** [review]
lgtm

**Pull request merged**